Error: 
  â”Œ tests/js_error/source_locations/expression/array/spread_get_iterator.js:3:11
  |
3 |     throw new Error();
  |           ^
Stack:
  at <anonymous> (tests/js_error/source_locations/expression/array/spread_get_iterator.js:3:11)
  at <global> (tests/js_error/source_locations/expression/array/spread_get_iterator.js:7:2)